src/features/Navigation/NavigationItems.tsx

```diff
@@ -1,26 +1,55 @@
+import { NavLink as RouterNavLink, useLocation } from 'react-router-dom';
 import styled from 'styled-components';
-import { NAVIGATION_HEIGHT, palette } from '@/components/variables';
+import { useState } from 'react';
 
-import { NavLink as RouterNavLink } from 'react-router-dom';
+import { NAVIGATION_HEIGHT, palette } from '@/components/variables';
+import { selectChats } from '../Chat/chatSlice';
+import { useAppSelector } from '@/store';
 
 export type NavigationItem = {
+  hasNotification?: boolean;
   text: string;
   url: string;
 };
 
-export const Item = ({ text, url }: NavigationItem) => (
-  <Link to={url} className="navbar-link">
-    {text}
-  </Link>
-);
+export const Item = ({ hasNotification, text, url }: NavigationItem) => {
+  const [isHovered, setIsHovered] = useState(false);
+  const isCurrent = useLocation().pathname === url;
```

Comment:
Here too I would say isCurrentLocation, so would be a bit more easier to the brainz

Reply:
I refactored this and the mobile component too!